src/helpers/Html/Html.jsx

```diff
@@ -82,7 +84,7 @@ class Html extends Component {
           <meta name="viewport" content="width=device-width, initial-scale=1" />
           <meta name="apple-mobile-web-app-capable" content="yes" />
           {/* Add the crossorigin while in development */}
-          {extractor.getLinkElements().map((elem) =>
+          {extractor.getLinkElements({ rel: 'preload' }).map((elem) =>
```

Comment:
I don't think this is needed, make sure to test in "production" mode by running "yarn build; yarn start:prod"

Reply:
I think it helps: https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content. I tested with the given commands. The score goes from 17 to 16 on my laptop when compared to Volto master, but it is WIP.